.github/workflows/vectors-matrix.yml

```
@@ -0,0 +1,68 @@
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with: { python-version: '3.x' }
+      - name: Install python deps
+        run: python -m pip install --upgrade pip blake3
+      - name: Python compute id
+        id: py
+        run: |
```

actionlint (1.7.8)

shellcheck reported issue in this script: SC2086:info:2:31: Double quote to prevent globbing and word splitting

(shellcheck)